config/crd/bases/sriovnetwork.openshift.io_sriovnetworknodestates.yaml

```diff
@@ -75,6 +75,8 @@ spec:
                   - pciAddress
                   type: object
                 type: array
+              isPolicySet:
```

Comment:
Please, add the similar change to https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator/crds/sriovnetwork.openshift.io_sriovnetworknodestates.yaml